src/applications/proxy-rewrite/tests/teamsite-header-footer.vrt.spec.js

```diff
@@ -0,0 +1,51 @@
+const puppeteer = require('puppeteer');
+
+const SCREEN_SIZES = [
+  ['full screen', { width: 1024, height: 800 }],
+  ['tablet screen', { width: 768, height: 1024 }],
+  ['mobile screen', { width: 375, height: 667 }],
+  ['small mobile screen', { width: 318, height: 500 }],
+];
+
+const HEADER = 'header';
+const FOOTER = '#footerNav';
+
+const TEAMSITES = [
+  ['Benefits', 'https://benefits.va.gov'],
+  ['Benefitswww', 'https://www.benefits.va.gov'],
+  ['CEM', 'https://www.cem.va.gov'],
+  ['Choose', 'https://www.choose.va.gov'],
+];
+
+describe.each(TEAMSITES)('%s', (sitename, url) => {
+  let browser;
+  let page;
+  jest.setTimeout(20000);
+
+  const compareScreenshots = async selector => {
+    const element = await page.$(selector);
+    await page.waitFor(1000);
+    const image = await element.screenshot();
+    expect(image).toMatchImageSnapshot();
+  };
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+    await page.goto(url);
+    await page.waitForSelector(FOOTER);
+    // hide foresee
+    await page.addStyleTag({ content: '.__acs { display: none !important; }' });
+  });
+
+  it.each(SCREEN_SIZES)('renders %s size', async (sreensizeName, viewport) => {
+    await page.setViewport(viewport);
+
+    await compareScreenshots(FOOTER);
```

Comment:
Could move these selectors into the table as well so we can easily test several parts of the pages. We don't have to test giant things like the header- we could test the things like buttons, etc.

Reply:
For now the image generator is numbering the image snapshots for the 1-FOOTER and 2-HEADER. Think I'll add this as a todo- would be better if we did a separate it statement for each screenshot so the images can be named with the site name + selector name.